zoo_project.py

```diff
@@ -34,35 +35,55 @@ class Cage:
     """
     cage_counter = 1
 
+    def add_predator(self, new_animal):
+        """Adds a predator to the cage
```

Comment:
Throughout this file, the formatting of the docstrings isn't following pep8 (https://www.python.org/dev/peps/pep-0008/#documentation-strings).